src/ext/java/org/opentripplanner/ext/bikerentalservicedirectory/GbfsUpdaterSourceConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.ext.bikerentalservicedirectory;
+
+import org.opentripplanner.updater.UpdaterDataSourceConfig;
+import org.opentripplanner.updater.UpdaterDataSourceParameters;
+
+public class GbfsUpdaterSourceConfig implements UpdaterDataSourceConfig {
```

Comment:
The `UpdaterDataSourceConfig` is still a bit "stange", but not part of this PR. I might look at it later.